__tests__/app.test.js

```diff
@@ -406,3 +406,33 @@ describe('/api/articles/:article_id/comments', () => {
 		});
 	});
 });
+
+describe('/api/comments/:comment_id', () => {
+	describe('204: DELETE:', () => {
+		it('responds with status 204 and a message', () => {
```

Comment:
You might want to change this description as there is no message being tested here